3rdparty/celt-0.7.0-build/celt-0.7.0-build.pro

```diff
@@ -71,13 +71,11 @@ unix {
 
 SOURCES *= bands.c celt.c cwrs.c entcode.c entdec.c entenc.c header.c kiss_fft.c kiss_fftr.c laplace.c mdct.c modes.c pitch.c psy.c quant_bands.c rangedec.c rangeenc.c rate.c vq.c
 
-CONFIG(debug, debug|release) {
-  CONFIG += console
-  DESTDIR = ../../debug
-}
-
-CONFIG(release, debug|release) {
-  DESTDIR = ../../release
-}
+# TARGET is the same in both versions of CELT, thus we have to specify
+# a separated folder to prevent objects collision between the two.
+OBJECTS_DIR = $$OBJECTS_ROOT/3rdparty/celt-0.7.0-build
```

Comment:
We don't do subdirs anymore, so drop `3rdparty`. Also, drop the `-build` suffix, since we use target names in all other cases but here -- and having "build" in the "target" seems odd.